File: web3rpc/rpc-specs/code-samples/python/klay/transaction/getTransactionReceipt.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

transactionHash = "0xaca5d9a1ed8b86b1ef61431b2bedfc99a66eaefc3a7e1cffdf9ff53653956a67"

w3 = Web3(Web3.HTTPProvider(host))
klay_response = w3.klay.get_transaction_receipt(transactionHash)

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/transaction/getTransactionReceipt.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");

  const transactionHash =
    "0x2781f2f57b2587f6d9ad80a9e5f60158439d2548eebbc23bd806ecb856fe724e";
  provider.klay
    .getTransactionReceipt(transactionHash, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
